### PR TITLE
zh-cn: init the translation of "Game Tutorials"

### PR DESCRIPTION
**Pull request by yin1999**
### Description

init the translation of "Game Tutorials"

### CODE REVIEW

File: files/zh-cn/games/tutorials/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+{{GamesSidebar}}
+
+本页面包含多个教程系列，重点介绍高效创建不同类型的 Web 游戏的不同工作流程。
+
+- [使用纯 JavaScript 实现 2D 打砖块游戏](/zh-CN/docs/Games/Tutorials/2D_Breakout_game_pure_JavaScript)
+  - : 在该手把手教程中，你将使用纯 JavaScript 来实现一个简单的打砖块游戏。在这个过程中，你将学习使用 {{htmlelement("canvas")}} 元素来实现基本的游戏机制，如渲染和移动图像、碰撞检测、控制机制以及获胜和失败状态。
+- [使用 Phaser 实现 2D 打砖块游戏](/zh-CN/docs/Games/Tutorials/2D_breakout_game_Phaser)
+  - : 在该手把手教程中，你将实现与之间的教程系列中相同的打砖块游戏，只不过这次你将使用 [Phaser](https://phaser.io/) HTML 游戏框架来实现。这里的想法是教授一些使用框架的基础知识（及优势），以及基本的游戏机制。
```
Comment (jasonren0403) [inline files/zh-cn/games/tutorials/index.md:13]:
```suggestion
  - : 在该手把手教程中，你将实现与之前的教程系列中相同的打砖块游戏，只不过这次你将使用 [Phaser](https://phaser.io/) HTML 游戏框架来实现。这里的想法是教授一些使用框架的基础知识（及优势），以及基本的游戏机制。
```

File: files/zh-cn/games/tutorials/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- [使用设备方向的 2D 迷宫游戏](/zh-CN/docs/Games/Tutorials/HTML5_Gamedev_Phaser_Device_Orientation)
+  - : 该教程展示如何使用 HTML，结合碰撞检测及在 {{htmlelement("canvas")}} 中放置精灵的基础知识来创建一个 2D 迷宫游戏。这是一个移动游戏，它使用[设备方向](/zh-CN/docs/Web/API/Device_orientation_events) 和[振动](/zh-CN/docs/Web/API/Vibration_API) **API** 来增强游戏体验，并使用 [Phaser](https://phaser.io/) 框架构建。
```
Comment (jasonren0403) [inline files/zh-cn/games/tutorials/index.md:15]: 精灵？

```suggestion
  - : 该教程展示如何使用 HTML，结合碰撞检测及在 {{htmlelement("canvas")}} 中放置精灵图的基础知识来创建一个 2D 迷宫游戏。这是一个移动端游戏，它使用[设备方向](/zh-CN/docs/Web/API/Device_orientation_events)和[振动](/zh-CN/docs/Web/API/Vibration_API) **API** 来增强游戏体验，并使用 [Phaser](https://phaser.io/) 框架构建。
```

Comment (yin1999) [reply files/zh-cn/games/tutorials/index.md:15]: https://docs.unity3d.com/cn/2018.1/Manual/class-SpriteRenderer.html

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**